scripts/genome_wide/create_window_pairs.py

```diff
@@ -74,7 +75,7 @@ def get_windows(carrays_dir, outDir, chrom_list, win, label_file_path, mode, npz
         n_r = 10 ** 5
         last_t = time()
         i = 1
-        padding = np.zeros(shape=(padding_len, n_channels), dtype=np.float32)
+        padding = np.zeros(shape=(padding_len, n_channels), dtype=np.half)
```

Comment:
Is this precision sufficient?

```
>>> np.finfo(np.float32).precision
6
>>> np.finfo(np.half).precision
3
``

Reply:
From the result on the test data it seems to be sufficient, but I need to try it on the real data.